exec/main.c

```diff
@@ -1198,6 +1289,7 @@ int main (int argc, char **argv, char **envp)
 					"        -f     : Start application in foreground.\n"\
 					"        -p     : Do not set realtime scheduling.\n"\
 					"        -r     : Set round robin realtime scheduling (default).\n"\
+					"        -R     : Do not try move corosync to root cpu cgroup\n" \
```

Comment:
I understand the aversion to #ifdef but I think here it's not very intrusive (because it's just in a load of printfs so doesn't obscure the meaning of the code) and it makes the usage message more meaningful on non-Linux platforms. If you really want to avoid the ifdef then at least put "(Linux only)" after the description.